shared/src/business/entities/Statistic.test.ts

```diff
@@ -271,15 +263,6 @@ describe('Statistic', () => {
       });
     });
 
-    it('should update the penalty in the penalties array', () => {
-      expect(statistic.penalties.length).toEqual(penaltyArrayLength);
-
-      statistic.updatePenalty(MOCK_UPDATED_PENALTY);
-
-      expect(statistic.penalties.length).toEqual(penaltyArrayLength);
-      expect(statistic.penalties[0]).toEqual(MOCK_UPDATED_PENALTY);
-    });
```

Comment:
`updatePenalty` was never called anywhere in the codebase, so I got rid of it and its test here.